docs/release-notes/RELEASE-NOTES-2.0.1.md

```diff
@@ -0,0 +1,106 @@
+# About this release
+
+[This][this-release] is a bugfix release.
+It contains:
```

Comment:
Maybe mention it will have *only* windows binaries as there are no related changes to the node but only to windows scripts.